tests/test_dataloader.py

```diff
@@ -462,6 +465,31 @@ def test_path(self):
         loader = dataloader.Loader()
         assert loader.load('/', Path) == Path('/')
 
+    def test_pattern_str(self):
+        loader = dataloader.Loader()
+        if sys.version_info[:2] <= (3, 8):
```

Comment:
This is not necessary, python 3.8 is the lowest supported version at this time.

Reply:
I could remove the `<` part if you like but when removing this entirely the tests fail on 3.8:
```
======================================================================
ERROR: test_pattern_bytes (tests.test_dataloader.TestCommonTypes)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/var/tmp/src/typedload/tests/test_dataloader.py", line 485, in test_pattern_bytes
    assert loader.load(br'[bc](at|ot)\d+', re.Pattern[bytes]) == re.compile(br'[bc](at|ot)\d+')
TypeError: 'type' object is not subscriptable

======================================================================
ERROR: test_pattern_str (tests.test_dataloader.TestCommonTypes)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/var/tmp/src/typedload/tests/test_dataloader.py", line 475, in test_pattern_str
    assert loader.load(r'[bc](at|ot)\d+', re.Pattern[str]) == re.compile(r'[bc](at|ot)\d+')
TypeError: 'type' object is not subscriptable
```